packages/babel-helper-split-export-declaration/src/index.js

```diff
@@ -0,0 +1,65 @@
+import * as t from "@babel/types";
+
+const splitExportDeclaration = exportDeclaration => {
```

Comment:
Style-wise, I find splitting up the `export default splitExportDeclaration;` and this harder to follow. Any reason not to use `export default function splitExportDeclaration(exportDeclaration) {` instead?

Reply:
Agreed, no idea why I didn't use a one-liner here. Gonna change it soon.